tasks/fh-build.js

```diff
@@ -243,174 +222,162 @@ module.exports = function(grunt) {
+            var cmdsString = grunt.config.get(test_type)
+            var cmdArray = cmdsString.split(',')
+            cmd = cmdArray.join(' && ')
           } else {
-            grunt.warn("Invalid or missing parameter to " + this.nameArgs);
+            grunt.log.warn('Skipping', grunt.task.current.nameArgs,
+              '-- invalid or missing parameter')
           }
 
-          return cmd;
+          return cmd
         }
       },
 
       'fh-report-cov': {
-        command: function() {
-          var cmd = './node_modules/.bin/istanbul report';
+        command: function () {
+          var cmd = './node_modules/.bin/istanbul report'
 
-          var argsArray = Array.prototype.slice.call(arguments);
-          argsArray.forEach(function(arg) {
-            cmd += ' ' + arg;
-          });
+          var argsArray = Array.prototype.slice.call(arguments)
+          argsArray.forEach(function (arg) {
+            cmd += ' ' + arg
+          })
 
-          return cmd;
+          return cmd
         }
-      },
-      'fh-nsp': {
-        command: 'PATH=node_modules/.bin:node_modules/grunt-fh-build/node_modules/.bin:$PATH nsp check'
       }
     }
-  });
-
+  })
 
-  var checkPlaceholderFileExists = function() {
+  var checkPlaceholderFileExists = function () {
     try {
-      require(path.resolve('config/ose-placeholders.js'));
-      return true;
+      require(path.resolve('config/ose-placeholders.js'))
+      return true
+    } catch (exception) {
+      grunt.log.debug('No placeholder file found for openshift 3 - continuing normally')
+      return false
     }
-    catch (exception) {
-      grunt.log.debug('No placeholder file found for openshift 3 - continuing normally');
-      return false;
-    }
-  };
+  }
 
   var runTestsForSingleFile = function (args) {
-    var testFile = args[0];
+    var testFile = args[0]
     if (!testFile) {
-      grunt.log.errorlns("Please specify test file to run: grunt fh:testfile:filename.js");
-      return;
+      grunt.log.errorlns('Please specify test file to run: grunt fh:testfile:filename.js')
+      return
     }
     // Task name (grunt file config variable)
-    var testType = "unit_single";
+    var testType = 'unit_single'
     // Task filename parameter (used as template in runtime script)
-    grunt.config.set("unit_test_filename", testFile);
+    grunt.config.set('unit_test_filename', testFile)
     for (var index = 1; index < args.length; index++) {
-      grunt.config.set("unit_test_param" + index, args[index]);
+      grunt.config.set('unit_test_param' + index, args[index])
     }
-    var cmdArray = grunt.config.get(testType);
+    var cmdArray = grunt.config.get(testType)
     if (!cmdArray) {
-      grunt.log.errorlns("No local config for single tests. Please define " + testType + " property in config.");
-      return;
+      grunt.log.errorlns('No local config for single tests. Please define ' + testType + ' property in config.')
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)